src/components/blocked-patron/blocked-modal/BlockedModal.tsx

```diff
@@ -15,9 +15,18 @@ const BlockedModal: FC<BlockedModalProps> = ({ blockedStatus }) => {
   const u = useUrls();
   const blockedPatronELinkUrl = u("blockedPatronELinkUrl", true);
   const { blockedModal } = getModalIds();
+  const [isJustLoggedIn, setJustLoggedIn] = useState<boolean>(false);
 
-  // If the user isn't actually blocked, don't render the modal.
-  if (!blockedStatus || blockedStatus === "") {
+  useEffect(() => {
+    const urlPrams = new URLSearchParams(window.location.search);
+    if (urlPrams.get("check_logged_in")) {
+      setJustLoggedIn(true);
+    }
+  }, []);
```

Comment:
Please use our [global url function
](https://github.com/danskernesdigitalebibliotek/dpl-react/blob/45e9d7b48c7763ae42648830e0e286d07c3d0860/src/core/utils/helpers/url.ts#L18).
Another thing I think you should make use of is a "tristate" version of `isJustLoggedIn` (`boolean | null`)  instead of `boolean`. In that way you can detect if the state has been "touched", and make sure not to rerun the functionality in useEffect on every load. I will show you in person.